codec-http/src/main/java/io/netty/handler/codec/http/cookie/ServerCookieDecoder.java

```diff
@@ -121,8 +121,14 @@ private ServerCookieDecoder(boolean strict) {
 
                     valueBegin = i;
                     // NAME=VALUE;
-                    int semiPos = header.indexOf(';', i);
-                    valueEnd = i = semiPos > 0 ? semiPos : headerLen;
+                    int delimiterPos = header.indexOf(';', i);
+                    if (rfc2965Style) {
```

Comment:
if `rfc2965Style` should we just do a loop which searches for the first index of `;` or `,` to avoid 2 `indexOf` calls and iterations (which may not find a `,` character)